### PR TITLE
Switch to Dart Sass, dropping support for Node Sass.

### PR DESCRIPTION
**Pull request by mirisuzanne**
momma sloth

Fixes: #153
Fixes: #167 
Fixes: #168 

## Description
- [x] Use Dart Sass & module syntax
- [x] Sass tests passing
- [x] Set Dart Sass as the default compiler
- [x] JS tests passing
- [x] fix Herman docs
- [x] re-organize modules
- [x] broken "contains" tests
- [x] missing utility tests

### CODE REVIEW

File: sass/true/_modules.scss [hunk 1]
```diff
@@ -65,57 +66,49 @@
...
-      $modules: $modules + ' :: ' + nth($module-list, $i);
+      $modules: $modules + ' :: ' + list.nth($module-list, $i);
     }
   }
 
-  @include _true-message('# Module: #{$modules}', 'comments');
+  @include messages.message('# Module: #{$modules}', 'comments');
 
-  $underline: '----------';
+  $underline: '--------';
```
Comment (jgerigmeyer) [inline sass/true/_modules.scss:96]: @mirisuzanne I'm, uh, not sure why this change was necessary to maintain the right length of underlines?

File: test/css/test.css [hunk 2]
```diff
@@ -711,3 +678,8 @@
 /*   ✔ [assert-equal] Linebreaks */
 /*  */
 /*  */
+.not-a-test {
+  break: please-no;
+}
```
Comment (jgerigmeyer) [inline test/css/test.css:683]: 🤷‍♂️

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,12 @@
 
 ## UNRELEASED
 
+- BREAKING: Make [Dart Sass](https://sass-lang.com/dart-sass) the default Sass
+  implementation
+- BREAKING: Drop support for node < 10
+- BREAKING: Rename `$true-terminal-output` setting to `$terminal-output`
```
Comment (jgerigmeyer) [inline CHANGELOG.md:8]: @mirisuzanne Let me know if I missed something here?

File: package.json [hunk 4]
```diff
@@ -18,28 +18,36 @@
...
-    "sassdoc": "^2.7.0",
-    "sassdoc-theme-herman": "^3.0.2"
+    "coveralls": "^3.0.11",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-prettier": "^3.1.2",
+    "jest": "^25.3.0",
+    "mocha": "^7.1.1",
+    "node-sass": "^4.13.1",
```
Comment (jgerigmeyer) [inline package.json:39]: @mirisuzanne Do we want to run the tests on both dart-sass and node-sass? Might be a good idea, if we claim to support both... Otherwise we can remove this node-sass dependency, I think.

Comment (mirisuzanne) [reply package.json:39]: It's good conceptually, but I also updated the internal syntax. So once we merge my changes, Dart-Sass will be required for the upgrade (until node-sass supports modules).

We could split out your changes from mine, if we want to release a minor update that supports Dart Sass without dropping node-sass support?

Comment (jgerigmeyer) [reply package.json:39]: @mirisuzanne Hm, okay -- I'm not sure a new release is required then. True _already_ supported both node-sass and dart-sass, right? But in that case I should make a few changes here to make it clear we're dropping node-sass support entirely.

Comment (mirisuzanne) [reply package.json:39]: ah, good point. Yeah, go for it.

Comment (jgerigmeyer) [reply package.json]: Ok, removed support for node-sass.

**Comment by jgerigmeyer** [comment]
[Update True to use Dart-Sass.](https://trello.com/c/4mxPZwPE/10-update-true-to-use-dart-sass)

File: package.json [hunk 5]
```diff
@@ -18,28 +18,36 @@
...
-    "css": "^2.2.1",
-    "lodash.find": "^4.6.0",
-    "lodash.foreach": "^4.5.0",
-    "lodash.last": "^3.0.0"
+    "chalk": "^4.1.0",
+    "css": "^3.0.0",
+    "lodash": "^4.17.19"
+  },
+  "peerDependencies": {
+    "sass": ">=1.23.0"
```
Comment (jgerigmeyer) [inline package.json:29]: @mirisuzanne I'm not really sure what version of `sass` we require, but this seemed like a bare minimum. 🤷

File: .sassdocrc [hunk 6]
```diff
@@ -1,11 +1,12 @@
-theme: 'herman'
+theme: herman
```
Comment (jgerigmeyer) [inline .sassdocrc:1]: @mirisuzanne I notice you had this pointing to your local Herman. Is there something missing in the published v3.2.0 that we need for True?

Comment (mirisuzanne) [reply .sassdocrc:1]: nope, probably just from testing before we released Herman

File: package.json (hunk 5 above)
Comment (mirisuzanne) [reply package.json:29]: Yeah, that's a good guess. I don't think we're using anything terribly advanced, so probably safe
